src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -89,13 +94,21 @@ public void storeMessage(Message message) {
     for (Entity entity : results.asIterable()) {
       try {
         String idString = entity.getKey().getName();
+        System.out.println("successfully read the message entity ID");
         UUID id = UUID.fromString(idString);
         String user = (String) entity.getProperty("user");
+        System.out.println("successfully read the message entity USER");
         String text = (String) entity.getProperty("text");
+        System.out.println("successfully read the message entity TEXT");
         long timestamp = (long) entity.getProperty("timestamp");
+        System.out.println("successfully read the message entity TIMESTAMP");
+        Float sentimentScore = (Float) entity.getProperty("sentimentScore");
```

Comment:
Will this work?

```suggestion
        Double sentimentScore = entity.getProperty("sentimentScore");
```

Reply:
OR maybe if it already returns as a float you dont have to cast it? Try this link to test and see what value the google cloud api returns you: https://medium.com/google-cloud/sentiment-analysis-using-google-cloud-machine-learning-552be9b9c39b and then debug from there?

Reply:
Thank you! Ill check out the link :)) 

So if I leave it as a double here, then I just control how many decimals are printed later using .2f or the equivalent of that?